apps/frontpage/app/docs/[[...slug]]/layout.tsx

```diff
@@ -78,17 +77,17 @@ export default async function Layout({
   children,
   params: { slug },
 }: PageProps) {
+  console.log('docs layout, slug', slug);
```

Comment:
@kylegach after we do some additional testing with this, we should remove the ' console.log ()` so that the user doesn't get overwhelmed by these messages, as they don't need to know what's happening as this is aimed for development purposes & testing. Wouldn't you agree?